cleverpush/src/main/java/com/cleverpush/inbox/InboxDetailActivity.java

```diff
@@ -164,12 +166,27 @@ private void init() {
   }
 
   private View createLayout(int layoutId) {
+    boolean isExceptionGenerate = false;
+    View layout = null;
     try {
-      View layout = activity.getLayoutInflater().inflate(layoutId, null);
-      layout.setOnClickListener(view -> dismiss());
-      return layout;
+      layout = activity.getLayoutInflater().inflate(layoutId, null);
     } catch (Exception exception) {
       Logger.e(TAG, exception.getLocalizedMessage());
+      isExceptionGenerate = true;
+    } finally {
+      try {
+        if (isExceptionGenerate) {
```

Comment:
same stuff for this, maybe we even want to but this whole try catch into a function not sure